queries/glimmer/highlights.scm

```diff
@@ -1,113 +1,84 @@
 ; === Tag Names ===
+
 ; Tags that start with a lower case letter are HTML tags
 ; We'll also use this highlighting for named blocks (which start with `:`)
 ((tag_name) @tag
-  (#lua-match? @tag "^:?[%l]"))
-
+  (#match? @tag "^(:)?[a-z]"))
 ; Tags that start with a capital letter are Glimmer components
 ((tag_name) @constructor
-  (#lua-match? @constructor "^%u"))
+  (#match? @constructor "^[A-Z]"))
 
-(attribute_name) @attribute
+(attribute_name) @property
 
 (string_literal) @string
-
```

Comment:
Don't make formatting changes; we enforce style via the format-queries script, just run that locally.

Reply:
this is literally a `cp` -- what I need to do is run your formatting tool in the tree-sitter-glimmer* repos

Reply:
Please do that before opening a PR.

Reply:
it _is_ a draft PR -- it wasn't meant to be ready for review. 
I open draft PRs to learn what CI requirements are in repos I'm not familiar with.

Reply:
no longer draft! :tada: